lib/app/modules/registration/views/registration_view.dart

```diff
@@ -0,0 +1,162 @@
+              SingleChildScrollView(
+                child: Padding(
+                  padding: const EdgeInsets.all(16.0),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      SizedBox(height: 10),
+                      Center(
+                        child: Text(
+                          'STUDENT DETAILS',
+                          style: Get.theme.kSubTitleTextStyle,
+                        ),
+                      ),
+                      SizedBox(height: 20),
+                      Container(
+                        padding: const EdgeInsets.all(16.0),
+                        decoration: BoxDecoration(
+                          boxShadow: [
+                            BoxShadow(
+                              blurRadius: 8,
+                              color: Get.theme.colorLevel0,
+                              spreadRadius: 5,
+                            )
+                          ],
+                          color: Get.theme.colorLevel2, // Match the container background color
+                          borderRadius: BorderRadius.circular(15),
+                        ),
+                        child: Column(
+                          children: [
+                            _buildTextField('NAME:'),
+                            SizedBox(height: 30),
+                            _buildTextField('REGISTRATION NO:'),
+                            SizedBox(height: 30),
+                            _buildDropdownButtonFormField('DESIGNATION:', ['VOLUNTEER', 'COMMITTEE MEMBER ', 'COMMITTEE HEAD','ORGANIZER']),
+                            SizedBox(height: 30),
+                            _buildTextField('DEPARTMENT:'),
+                            SizedBox(height: 30),
+                            _buildDropdownButtonFormField('DIVISION:', ['A', 'B', 'C']),//IDK what to put in designation
+                            SizedBox(height: 30),
+                            _buildTextField('NET ID:'),
+                            SizedBox(height: 30),
+                            _buildTextField('Ph no:'),
+                            SizedBox(height: 30),
+
+                          ],
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+              ElevatedButton.icon(
+                  onPressed: () {},
+                  label: Text('ENROLL', style: Get.theme.kSmallTextStyle,),
+                  style: ElevatedButton.styleFrom(
+                    backgroundColor: Get.theme.colorLevel1,
+                  ),
+                  icon: Icon(Icons.arrow_circle_right_rounded,
+                    color: Get.theme.colorLevel4,
+                  )
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+  Widget _buildTextField(String labelText) {
+    return Container(
+      decoration: BoxDecoration(
```

Comment:
move this _buildTextFieldand and  _builDropDownButtonFormField to different folder, as told in previous pr as well.